scenarios/evaluate/AI_RedTeaming/AI_RedTeaming.ipynb

```diff
@@ -0,0 +1,447 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%python -m venv .venv\n",
+    "# %source .venv/bin/activate # If using Mac/Linux OS\n",
+    "%.venv/Scripts/activate # If using Windows OS\n",
+    "%pip install uv"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# install the packages\n",
+    "%uv pip install azure-ai-evaluation[redteam] termcolor==2.5.0 azure-identity openai"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set Up Your Environment Variables\n",
+    "\n",
+    "Set the following variables for use in this notebook. These variables connect to your Azure resources and model deployments.\n",
+    "\n",
+    "**Note:** You can find these values in your Azure AI Foundry project or Azure OpenAI resource."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For reference, here's an example of what your populated environment variables should look like:\n",
+    "\n",
+    "```\n",
+    "# Azure OpenAI\n",
+    "AZURE_OPENAI_API_KEY=\"your-api-key-here\"\n",
+    "AZURE_OPENAI_ENDPOINT=\"https://endpoint-name.openai.azure.com/openai/deployments/deployment-name/chat/completions\"\n",
+    "AZURE_OPENAI_DEPLOYMENT_NAME=\"gpt-4\"\n",
+    "AZURE_OPENAI_API_VERSION=\"2023-12-01-preview\"\n",
+    "\n",
+    "# Azure AI Project\n",
+    "AZURE_SUBSCRIPTION_ID=\"12345678-1234-1234-1234-123456789012\"\n",
+    "AZURE_RESOURCE_GROUP_NAME=\"your-resource-group\"\n",
+    "AZURE_PROJECT_NAME=\"your-project-name\"\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# Azure AI Project information\n",
+    "azure_ai_project = {\n",
+    "    \"subscription_id\": os.environ.get(\"AZURE_SUBSCRIPTION_ID\"),\n",
+    "    \"resource_group_name\": os.environ.get(\"AZURE_RESOURCE_GROUP_NAME\"),\n",
+    "    \"project_name\": os.environ.get(\"AZURE_PROJECT_NAME\"),\n",
+    "}\n",
+    "\n",
+    "# Azure OpenAI deployment information\n",
+    "azure_openai_deployment = os.environ.get(\"AZURE_OPENAI_DEPLOYMENT\")  # e.g., \"gpt-4\"\n",
+    "azure_openai_endpoint = os.environ.get(\n",
+    "    \"AZURE_OPENAI_ENDPOINT\"\n",
+    ")  # e.g., \"https://endpoint-name.openai.azure.com/openai/deployments/deployment-name/chat/completions\"\n",
+    "azure_openai_api_key = os.environ.get(\"AZURE_OPENAI_API_KEY\")  # e.g., \"your-api-key\"\n",
+    "azure_openai_api_version = \"2023-12-01-preview\"  # Use the latest API version"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Configuration\n",
+    "The AI Red Teaming Agent requires an Azure AI Foundry project configuration and Azure credentials. Your project configuration will be used to log red teaming scan results after the run is finished.\n",
```

Comment:
the project configuration is the cell above, I missed this. can you move this markdown up one?